src/source_clang.cc

```diff
@@ -135,16 +138,16 @@ void Source::ClangViewParse::parse_initialize() {
                   parsed=true;
                   set_status("");
                 }
-                parse_mutex.unlock();
+                parse_lock.unlock();
```

Comment:
To be honest, if I wasn't 100% sure if the code would be equivalent, I took the safer route of just locking and unlocking the `std::unique_lock` (with `std::defer_lock`, which makes `std::unique_lock` not lock immediately in the constructor). This would make it 100% equivalent, with the added benefit of being exception safe.